src/Fable.Transforms/Replacements.fs

```diff
@@ -1215,7 +1215,7 @@ let operators (com: ICompiler) (ctx: Context) r t (i: CallInfo) (thisArg: Expr o
     // KeyValuePair is already compiled as a tuple
     | ("KeyValuePattern"|"Identity"|"Box"|"Unbox"|"ToEnum"), [arg] -> TypeCast(arg, t) |> Some
     // Cast to unit to make sure nothing is returned when wrapped in a lambda, see #1360
-    | "Ignore", [arg]  -> TypeCast(arg, Unit) |> Some
+    | "Ignore", [arg] -> Helper.CoreCall("Util", "ignore", Unit, [arg], ?loc=r) |> Some
```

Comment:
Can we use `($0, void 0)` as I just realized [Fable 1 was doing](https://github.com/fable-compiler/Fable/commit/98fa2c177f40dcaaa9003917041ae4ef97a92888)? I would like to avoid the extra call for all `ignore`s.

Reply:
@alfonsogarciacaro Sure, I see you already put that in, I'll open an issue if I see something break.